---
layout: post
title:  "VS ribozyme"
date:   2022-04-18T14:25:52-05:00
author: Chichau Miao
categories: ribozyme
---

#### click into different sections:

- [Timeline](#timeline)
- [Description](#description)
- [Structure](#structure)
- [Reference](#reference)

**Rfam ID:** na

## Timeline

***
<html lang="zh-cn">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title></title>

</head>
<style>
.timeline {
  list-style: none;
  padding: 10px 0 10px;
  position: relative;
}
.timeline:before {
  top: 0;
  bottom: 0;
  position: absolute;
  content: " ";
  width: 3px;
  background-color: #eeeeee;
  left: 50%;
  margin-left: -1.5px;
}
.timeline > li {
  margin-bottom: 10px;
  position: relative;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li:before,
.timeline > li:after {
  content: " ";
  display: table;
}
.timeline > li:after {
  clear: both;
}
.timeline > li > .timeline-panel {
  width: 46%;
  float: left;
  border: 1px solid #d4d4d4;
  border-radius: 2px;
  padding: 5px;
  position: relative;
  -webkit-box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
  box-shadow: 0 1px 6px rgba(0, 0, 0, 0.175);
}
.timeline > li > .timeline-panel:before {
  position: absolute;
  top: 10px;
  right: -12px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #ccc;
  border-right: 0 solid #ccc;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-panel:after {
  position: absolute;
  top: 10px;
  right: -10px;
  display: inline-block;
  border-top: 10px solid transparent;
  border-left: 10px solid #fff;
  border-right: 0 solid #fff;
  border-bottom: 10px solid transparent;
  content: " ";
}
.timeline > li > .timeline-badge {
  color: #fff;
  width: 25px;
  height: 25px;
  line-height: 40px;
  font-size: 1.4em;
  text-align: center;
  position: absolute;
  top: 16px;
  left: 48.5%;
  margin-left: 0px;
  background-color: #999999;
  z-index: 100;
  border-top-right-radius: 50%;
  border-top-left-radius: 50%;
  border-bottom-right-radius: 50%;
  border-bottom-left-radius: 50%;
}
.timeline > li.timeline-inverted > .timeline-panel {
  float: right;
}
.timeline > li.timeline-inverted > .timeline-panel:before {
  border-left-width: 0;
  border-right-width: 15px;
  left: -15px;
  right: auto;
}
.timeline > li.timeline-inverted > .timeline-panel:after {
  border-left-width: 0;
  border-right-width: 14px;
  left: -14px;
  right: auto;
}
.timeline-badge.primary {
  background-color: #2e6da4 !important;
}
.timeline-badge.success {
  background-color: #3f903f !important;
}
.timeline-badge.warning {
  background-color: #f0ad4e !important;
}
.timeline-badge.danger {
  background-color: #d9534f !important;
}
.timeline-badge.info {
  background-color: #5bc0de !important;
}
.timeline-title {
  margin-top: 0;
  color: inherit;
}
.timeline-body > p,
.timeline-body > ul {
  margin-bottom: 0;
  padding-bottom: 0;
}
.timeline-body > p + p {
  margin-top: 0px;
}

</style>

<!--<div class="container" width: 1026px></div>-->
    <ul class="timeline">
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">1990</a>  Discovery</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">1995</a>  Secondary structure</h4>
            </div>
            <div class="timeline-body">
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2001</a>  A730 loop is important.</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2001</a>  NMR structure of the isolated substrate helix</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2002</a>  A756 is critical for catalysis</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2005</a>  NMR structure of SL5</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2007</a>  G638 is critical for catalysis</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2008</a>  SAXS-deriverd structure</h4>
            </div>
            <!--<div class="timeline-body">
              <p>SAXS-deriverd structure</p>
            </div>-->
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2011</a>  NMR structure of A730 loop</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2014</a>  NMR structure of the III-IV-V three-way junction</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2014</a>  NMR structure of the II-III-VI three-way junction</h4>
            </div>
          </div>
        </li>
        <li class="timeline-inverted">
          <div class="timeline-badge primary"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"> <a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2015</a>  Crystal structure</h4>
            </div>
          </div>
        </li>
        <li>
          <div class="timeline-badge warning"></div>
          <div class="timeline-panel">
            <div class="timeline-heading">
              <h4 class="timeline-title"><a href="https://www.ncbi.nlm.nih.gov/pubmed/14691255">2020</a> Catalytic mechanism</h4>
            </div>
          </div>
        </li>
    </ul>

</html>


## Description

***

<font size=4>Varkud satellite (VS) ribozyme is the largest known nucleolytic ribozyme and found to be embedded in VS RNA. VS RNA is a long non-coding RNA exists as a satellite RNA and is found in mitochondria of Varkud-1C and few other strains of Neurospora. VS ribozyme contains features of both catalytic RNAs and group 1 introns. VS ribozyme has both cleavage and ligation activity and can perform both cleavage and ligation reactions efficiently in the absence of proteins. VS ribozyme undergo horizontal gene transfer with other Neurospora strains. VS ribozymes have nothing in common with other nucleolytic ribozymes.</font><br>

<font size=4>VS RNA has a unique primary, secondary, and tertiary structure. The secondary structure of the VS ribozyme consists of six helical domains. Stem loop I forms the substrate domain while stem-loop II-VI forms the catalytic domain. When these 2 domains are synthesized in vitro separately, they can perform the self-cleavage reaction by trans-acting. The substrate binds into a cleft which is made by two helices. The likely active site of the ribozyme is a very important nucleotide A756. The A730 loop and A756 nucleotide are critical to its function since they participate in the phosphoric transfer chemistry activity of the ribozyme.</font>

## Structure

***
> 2D representation

<table><tr>
<td><embed src="http://localhost:4000/images/VSD1.svg" style="display:block;width:500px;height:400px" border=0 /></td>
<!--<td><img src="http://localhost:4000/images/VS_ribozymePic/VS2D.svg" alt="drawing" style="height:400px" border=0></td>-->
<td>
<link rel="stylesheet" type="text/css" href="fornac.css" media="screen" />

This is an RNA container.
<div id='rna_ss'> </div>
This is after the RNA container.
<meta charset="utf-8">
    <script type='text/javascript' src='{{ site.url }}{{ site.baseurl }}/js/jquery.js'></script>
    <script type='text/javascript' src='{{ site.url }}{{ site.baseurl }}/js/d3.js'></script>
    <script type='text/javascript' src='{{ site.url }}{{ site.baseurl }}/js/fornac.js'></script>
  
    <script type='text/javascript'>
        var container = new FornaContainer("#rna_ss", {'applyForce': false});

        var options = {'structure': '((..((....)).(((....))).))',
                       'sequence':             'CGCUUCAUAUAAUCCUAAUGACCUAU'};

        container.addRNA(options.structure, options);
    </script>
<!--<iframe src="http://nibiru.tbi.univie.ac.at/forna/forna.html?id=fasta&file=%3Ecircular_rna\nCUGCUCCACGCAAGGAGGUGGACUUAAGCGGCUCAUCCGGGUCUGCGAUAUCCACUGCGCGGUAUGCGCUCGCGAGUUCGAAUCUCGUCGCCAGUACACUGACUUCACUGGCGUGUCCGAGUGGUUAGGCAA\n..(((((((....(((((((((.....(((((((....))).))))....))))))((((.....))))..(((((.......)))))(((((((...........)))))))..)))..))))...)))..*" align="center" height="450px" width="450px" seamless='seamless' frameBorder="0" AllowFullScreen></iframe>-->
</td>
</tr></table><br>


> 3D visualisation

<table><tr>
<td><img src="http://localhost:4000/images/VS_ribozymePic/VS3d.png" alt="drawing" style="height:350px" border=0></td>
<td>
<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>PDBe Molstar</title>
    <!-- Molstar CSS & JS -->
    <link rel="stylesheet" type="text/css" href="https://www.ebi.ac.uk/pdbe/pdb-component-library/css/pdbe-molstar-3.0.0.css">
    <script type="text/javascript" src="https://www.ebi.ac.uk/pdbe/pdb-component-library/js/pdbe-molstar-plugin-3.0.0.js"></script>
    <!--<script src="{{ site.url }}{{ site.baseurl }}/js/pdbe-molstar-plugin-1.2.1.js"></script>-->
  <style>
      * {
          margin: 0;
          padding: 0;
          box-sizing: border-box;
      }
      .msp-plugin ::-webkit-scrollbar-thumb {
          background-color: #474748  !important;
      }
      .viewerSection {
        padding-top: 0px;
      }
      .controlsSection {
        width: 300px;
        float: margin-right;
        padding: 10px 0 0 0px;
        margin-right: 10px;
      }
      #myViewer{
        width:450px;
        height: 450px;
        position:relative;  
      }
      
  </style>
  </head>
  <body>
      <div class="controlsSection">
      <div class="controlBox">
        <button onclick="viewerInstance.canvas.setBgColor({r:255, g:255, b:255})">White</button>&nbsp;
        <button onclick="viewerInstance.canvas.setBgColor({r:0, g:0, b:0})">Black</button><br>
      </div>
    </div>
    <div class="viewerSection">
    <!-- Molstar container -->
    <div id="myViewer"></div>
    </div>
  <script>
    //Create plugin instance
    var viewerInstance = new PDBeMolstarPlugin();
    //Set options (Checkout available options list in the documentation)
    var options = {
      moleculeId: '4r4v',
      expanded: false,
      expanded: false,
      hideControls: true
      }
    var viewerContainer = document.getElementById('myViewer');
    viewerInstance.render(viewerContainer, options);
  </script>
  </body>
</html></td>
</tr></table><br>


> catalytic centre

<table><tr>
<td><img src="http://www.ribocentre.org/images/VS_ribozymePic/VSm1.png" alt="drawing" style="height:350px" border=0></td>
<td><img src="http://www.ribocentre.org/images/VS_ribozymePic/VSm2.png" alt="drawing" style="height:400px" border=0></td>
</tr></table><br>

## Reference

***

**A site-specific self-cleavage reaction performed by a novel RNA in neurospora mitochondria.**<br>
Saville, B. J. and R. A. Collins <br>
[Cell 61(4): 685-696.(1990)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>


**A secondary-structure model for the self-cleaving region of Neurospora VS RNA.**<br>
Beattie, T. L., J. E. Olive and R. A. Collins <br>
[Proc Natl Acad Sci U S A 92(10): 4686-4690.(1995)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>


**The A730 loop is an important component of the active site of the VS ribozyme.**<br>
Lafontaine, D. A., T. J. Wilson, D. G. Norman and D. M. Lilley <br>
[J Mol Biol 312(4): 663-674.(2001)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>


**A pH controlled conformational switch in the cleavage site of the VS ribozyme substrate RNA.**<br>
Flinders, J. and T. Dieckmann <br>
[J Mol Biol 308(4): 665-679.(2001)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Functional Group Requirements in the Probable Active Site of the VS Ribozyme.**<br>
Lafontaine, D. A., T. J. Wilson, Z.-Y. Zhao and D. M. J. Lilley <br>
[Journal of Molecular Biology 323(1): 23-34.(2002)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Nuclear magnetic resonance structure of the Varkud satellite ribozyme stem-loop V RNA and magnesium-ion binding from chemical-shift mapping.**<br>
Campbell, D. O. and P. Legault <br>
[Biochemistry 44(11): 4157-4170.(2005)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**A guanine nucleobase important for catalysis by the VS ribozyme.**<br>
Wilson, T. J., A. C. McLeod and D. M. Lilley <br>
[EMBO J 26(10): 2489-2500.(2007)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**The complete VS ribozyme in solution studied by small-angle X-ray scattering.**<br>
Lipfert, J., J. Ouellet, D. G. Norman, S. Doniach and D. M. Lilley <br>
[Structure 16(9): 1357-1367.(2008)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Nucleobase-mediated general acid-base catalysis in the Varkud satellite ribozyme.**<br>
Wilson, T. J., N. S. Li, J. Lu, J. K. Frederiksen, J. A. Piccirilli and D. M. Lilley <br>
[Proc Natl Acad Sci U S A 107(26): 11751-11756.(2010)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**NMR structure of the A730 loop of the Neurospora VS ribozyme: insights into the formation of the active site.**<br>
Desjardins, G., E. Bonneau, N. Girard, J. Boisbouvier and P. Legault <br>
[Nucleic Acids Res 39(10): 4427-4437.(2011)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Nuclear magnetic resonance structure of the III-IV-V three-way junction from the Varkud satellite ribozyme and identification of magnesium-binding sites using paramagnetic relaxation enhancement.**<br>
Bonneau, E. and P. Legault <br>
[Biochemistry 53(39): 6264-6275.(2014)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**The NMR structure of the II-III-VI three-way junction from the Neurospora VS ribozyme reveals a critical tertiary interaction and provides new insights into the global ribozyme structure.**<br>
Bonneau, E., N. Girard, S. Lemieux and P. Legault <br>
[RNA 21(9): 1621-1632.(2014)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Crystal structure of the Varkud satellite ribozyme.**<br>
Suslov, N. B., S. DasGupta, H. Huang, J. R. Fuller, D. M. Lilley, P. A. Rice and J. A. Piccirilli <br>
[Nat Chem Biol 11(11): 840-846.(2015)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Structural Basis for Substrate Helix Remodeling and Cleavage Loop Activation in the Varkud Satellite Ribozyme.**<br>
DasGupta, S., N. B. Suslov and J. A. Piccirilli <br>
[J Am Chem Soc 139(28): 9591-9597.(2017)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>

**Confluence of theory and experiment reveals the catalytic mechanism of the Varkud satellite ribozyme.**<br>
Ganguly, A., B. P. Weissman, T. J. Giese, N. S. Li, S. Hoshika, S. Rao, S. A. Benner, J. A. Piccirilli and D. M. York<br>
[Nat Chem 12(2): 193-201.(2020)]((https://www.ncbi.nlm.nih.gov/pubmed/14690435/))<br><br>


